### DISCUSSION TITLE
my license not work

### DISCUSSION THREAD

**Discussion by kisslevente2003** [aclap-dev/video-downloadhelper#374] [Help and Q&A]
### Your question

I purchased a Lifetime Premium VDH , Chrome license on   03/31/2020 .  
Ident number of transaction is
CWS.6281-3372-6413-45062
 Somehow my license disappeared and I can not find it anywhere. 

Please help.
Thank you

### Addon & CoApp version and details

Video DownloadHelper 8.2.2.9 (stable)
Video DownloadHelper
Version 8.2.2.9 (stable)
Browser locale: en-US
Production build
Built on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google
Platform Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
Found companion app: VdhCoApp 2.0.19
Companion app binary: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by jcfrog** [comment]
hi,
please send us your email address at licenses@downloadhelper.net with this transaction number.
